app/graphql/mutations/wallet_transactions/create.rb

```diff
@@ -16,7 +16,7 @@ class Create < BaseMutation
       type Types::WalletTransactions::Object.collection_type
 
       def resolve(**args)
-        result = ::WalletTransactions::CreateService.new.create(
+        result = ::WalletTransactions::CreateService.new(context[:current_user]).call(
```

Comment:
Same comment as above: 
```suggestion
        result = ::WalletTransactions::CreateService.call(context[:current_user]
```